### DISCUSSION TITLE
arrays tp3

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#110] [Preguntas y respuestas] [answered]
cuando defino la funcion para el array fuera del main, donde tengo que pedir los datos al usuario?, en el main o en la funcion con el array?

**Comment by martinvilu** [comment, marked as answer]
En una función creada para ese propósito, es uno de los ejercicios.

El tema de hacer los printf y scanf en el main es para que la función cumpla con su trabajo con todos los recursos dados solo por sus argumentos.
